### PR TITLE
Coffe Loop

### PR DESCRIPTION
**Pull request by Edvard777**

### CODE REVIEW

File: main/src/OnACoffeeLoop.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    static String back = "back";
+
+
+    public static void action ()  // method action asks to user what he can do (buy, fill, take, see remaining,or exit)
+    {
+        System.out.println("Write action (buy, fill, take, remaining, exit):");
+        Scanner scan = new Scanner(System.in);
+        String actionType = scan.nextLine();
+
+        if (buy.equals(actionType)) {
```
Comment (nsimonyan) [inline main/src/OnACoffeeLoop.java]: @Edvard777 you can use switch case here

File: main/src/OnACoffeeLoop.java [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            exit();
+        }
+    }
+
+    public static void buy ()   //method buy asks to input which coffee wants to make,or if change the mind type back
+    {
+        System.out.println("What do you want to buy? 1 - espresso, 2 - latte, 3 - cappuccino, back - to main menu:");
+        Scanner scaner = new Scanner(System.in);
+        String coffeeType = scaner.nextLine();
+        if(espresso.equals(coffeeType))
```
Comment (nsimonyan) [inline main/src/OnACoffeeLoop.java]: @Edvard777 the same is here

File: main/src/OnACoffeeLoop.java [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        {
+            System.out.println("Sorry, not enough coffee beans!");
+        }
+        else if(disCups==0)
+        {
+            System.out.println("Sorry, not enough disposable cups");
+        }
+        action();
+    }
+    public static void latte ()     //method latte which checks if available ingredients, makes a coffee,
```
Comment (nsimonyan) [inline main/src/OnACoffeeLoop.java]: @Edvard777 will be better to have comments on top of the method with /** comment **/

File: main/src/OnACoffeeLoop.java [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    static String take = "take";
+    static String espresso = "1";
+    static String latte = "2";
+    static String cappuccino = "3";
+    static String exit = "exit";
+    static String remaining = "remaining";
+    static String back = "back";
+
+
+    public static void action ()  // method action asks to user what he can do (buy, fill, take, see remaining,or exit)
```
Comment (nsimonyan) [inline main/src/OnACoffeeLoop.java]: @Edvard777 why you don't use parameters , and making everything class level

File: main/src/OnACoffeeLoop.java (hunk 1 above)
Comment (Edvard777) [reply main/src/OnACoffeeLoop.java]: done

File: main/src/OnACoffeeLoop.java (hunk 2 above)
Comment (Edvard777) [reply main/src/OnACoffeeLoop.java]: done

File: main/src/OnACoffeeLoop.java (hunk 3 above)
Comment (Edvard777) [reply main/src/OnACoffeeLoop.java]: done

File: main/src/OnACoffeeLoop.java (hunk 4 above)
Comment (Edvard777) [reply main/src/OnACoffeeLoop.java]: done

**Review by nsimonyan — APPROVED** [review]
Congratulations !!!

**Pull request merged**